truffleruby/src/main/java/org/truffleruby/language/arguments/RunBlockKWArgsHelperNode.java

```diff
@@ -51,7 +51,7 @@ public Object execute(VirtualFrame frame) {
                 "Truffle.load_arguments_from_array_kw_helper(array, kwrest_name, binding)",
                 "array", array,
                 "kwrest_name", kwrestName,
-                "binding", Layouts.BINDING.createBinding(coreLibrary().getBindingFactory(), frame.materialize()));
+                "binding", Layouts.BINDING.createBinding(coreLibrary().getBindingFactory(), frame.materialize(), null));
```

Comment:
I think it's better to use `BindingNodes.createBinding(getContext(), frame.materialize())`, it avoids leaking details in all callers.